### PR TITLE
changed xml to entree builder

### PR DESCRIPTION
**Pull request by noah427**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -88,46 +107,85 @@ client.on("message", msg => {
...
+//       type,
+//       msg.author.tag,
+//       body,
+//       tags.split(",")
+//     );
+
+//     return entry;
+//   }
+// }
+
```
Comment (arnopensource) [inline main.js]: No need to leave it here
It is in the git history

File: main.js [hunk 2]
```diff
@@ -3,22 +3,41 @@ const storage = require("./storage/storageManager");
...
-
-    if (entry != "bad") {
-      storage.write(entry);
+  if (onGoingEntrees.has(msg.author.id)) {
+    infoGather(onGoingEntrees.get(msg.author.id), msg.content);
+  }
 
-      msg.reply("your entry was added to the repldex");
-    }
+  if (msg.content === "?writeEntree") {
```
Comment (arnopensource) [inline main.js]: What's going on if the user sends `?writeEntree` when he is in the info gathering phase ?

File: main.js [hunk 3]
```diff
@@ -3,22 +3,41 @@ const storage = require("./storage/storageManager");
...
+  // if (msg.channel.id == "605862116808720416") {
+  //   let entry = parseEntry(msg);
+
+  //   if (entry != "bad") {
+  //     storage.write(entry);
+
+  //     msg.reply("your entry was added to the repldex");
+  //   }
+  // }
+
```
Comment (arnopensource) [inline main.js]: Commented code
See VVV

**Comment by arnopensource** [comment]
And btw, we replaced Entree by Entry in the code x)

File: main.js (hunk 2 above)
Comment (arnopensource) [reply main.js]: I think there is not problem with the map, but is it clear for the user that the process restarted ?

**Comment by arnopensource** [comment]
Check for DMs ?

**Comment by noah427** [comment]
The issues listed above have been resolved... and then some more we found along the way

File: main.js [hunk 4]
```diff
@@ -3,22 +3,34 @@ const storage = require("./storage/storageManager");
 const discord = require("discord.js");
 const client = new discord.Client();
 const config = require("./config");
-const cheerio = require("cheerio");
+const entryBuilder = require("./entryBuilder");
 
-client.on("ready", () => console.log("YES CHEF"));
+client.on("ready", () => console.log("YES CHEF")); //what is this btw
```
Comment (arnopensource) [inline main.js:8]: We can change it to `Bot ready` if you want

**Review by arnopensource — COMMENTED** [review]
No content verification ?
Can you create a entry with no name ?

**Pull request merged**